### PR TITLE
fixed email bug with face recognision login

### PR DESCRIPTION
**Pull request by neringga**

### CODE REVIEW

File: VirtualLibrary/BookReturnWarning.cs [hunk 1]
```diff
@@ -9,22 +10,26 @@ public bool SendWarningEmail(string userEmail, DateTime returnTime, string title
...
+                EmailCredentials emailCredentials = new EmailCredentials();
+                SmtpClient SmtpServer = new SmtpClient(Constants.smtpServer);
+                MailMessage mail = new MailMessage
+                {
+                    From = new MailAddress(Constants.senderEmail),
+                    Body = Constants.warningText +
+                    Environment.NewLine + author + " " + title + " " + Constants.warningText2 + " " +
+                    returnTime.ToString(),
+                    Subject = Constants.subjectEmail,
+                };
```
Comment (austeja) [inline VirtualLibrary/BookReturnWarning.cs]: I would maybe extract the email creation into a separate method

File: VirtualLibrary/DataSources/Data/Constants.cs [hunk 2]
```diff
@@ -10,9 +10,13 @@ static class Constants
     {
         public const string senderEmail = "nergei87@gmail.com";
```
Comment (austeja) [inline VirtualLibrary/DataSources/Data/Constants.cs]: Such configuration things could go into email-details file that is already used for holding the credentials, o ever better - an app settings file

File: VirtualLibrary/DataSources/Data/TextFile.cs [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+namespace VirtualLibrary.DataSources.Data
+{
+    class TextFile
+    {
+        public List<string> ReadTextFile(string fileName)
+        {
+            string line;
+            List<string> textList = new List<string>();
+            StreamReader file = new StreamReader( Path.Combine(Directory.GetParent(
```
Comment (viktorija-kondratjevaite) [inline VirtualLibrary/DataSources/Data/TextFile.cs]: Maybe use named arguments here? 
Just for visuals and better readability

File: VirtualLibrary/DataSources/Data/TextFile.cs [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+                Directory.GetCurrentDirectory()).Parent.FullName, fileName));
```
Comment (viktorija-kondratjevaite) [inline VirtualLibrary/DataSources/Data/TextFile.cs]: Maybe use named arguments here?
Just for the visuals and better readability

**Pull request merged**